upstairs/src/lib.rs

```diff
@@ -1038,6 +1038,7 @@ async fn cmd_loop(
                  * To keep things alive, initiate a ping any time we have
                  * been idle for (TBD) seconds.
                  */
+                println!("[{}] send ping", up_coms.client_id);
```

Comment:
Just making sure you meant to include a new `println!` :)

Reply:
No, no.  This was debug cruft along with the updates around Ping on the downstairs side.  I'll remove both.

Reply:
Removed!